From cd53f8bef787fc6c3790db8d31f93916eae7be71 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 15 May 2018 10:28:43 -0700
Subject: [PATCH] clang: scan-view needs python 2.x

Some distributions e.g. archlinux have switched to pointing
python to python3, therefore its better to be specific about
python version needed.

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 clang/tools/scan-view/bin/scan-view | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clang/tools/scan-view/bin/scan-view b/clang/tools/scan-view/bin/scan-view
index 6165432e7af..04ad518a60c 100755
--- a/clang/tools/scan-view/bin/scan-view
+++ b/clang/tools/scan-view/bin/scan-view
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 from __future__ import print_function
 
